creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a service that allows platforms to manage payments on behalf of their users. Here's an example to illustrate how it works:
+>
+> Imagine you are an event organizer using Moongate to host and manage your events. You want to sell tickets online and receive payments from attendees. Instead of setting up your own payment processing system, you can use Stripe Connect. Moongate, as the platform, handles the payment processing for you. This means:
+>
+> **Moongate:** Manages the transactions, ensuring payments from ticket sales go through smoothly. <br />
```

Comment:
I think you will have to rewrite this whole section as I think it is misleaded here.
The basic thing is stripe connect removes the hassle of having organizers to ask payout from us they can directly receive their payments and we receive the platform fees